js/app.js

```diff
@@ -135,42 +177,93 @@ function handleUpdateRateForm(event) {
     .value.trim()
     .toUpperCase();
 
-  // Get the value of the new exchange rate from the input field and convert it to a float
+  // Get the value of the new exchange rate from the input field
   const newExchangeRate = parseFloat(
     document.getElementById("update-exchange-rate").value
   );
 
-  // Find the rate object for the base currency
-  const rateObject = currencyRates.find((rate) => rate.base === baseCurrency);
+  // Find the rate
+  const rate = currencyRates.find(
```

Comment:
You could create a function out of this `.find` and re-use it in `handleSearchForm`.